blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -84,9 +85,9 @@ export class PipelinePage extends Component {
                     </Title>
                     }
                     <PageTabs base={baseUrl}>
-                        <TabLink to="/activity">Activity</TabLink>
-                        <TabLink to="/branches">Branches</TabLink>
-                        <TabLink to="/pr">Pull Requests</TabLink>
+                        <TabLink to="/activity">t('bo.dashboard.activity')</TabLink>
+                        <TabLink to="/branches">t('bo.dashboard.branches')</TabLink>
+                        <TabLink to="/pr">t('bo.dashboard.pr')</TabLink>
```

Comment:
I am in doubt since I saw many examples having a prefix.  see e.g. https://github.com/jenkinsci/ssh-slaves-plugin/blob/master/src/main/resources/hudson/plugins/sshslaves/Messages.properties 

```
SSHLauncher.RemoteFSDoesNotExist
```

If you use multiple bundles you could have the same key, however in i18next you could use ns prefixes to get a specific key.